projects/Mallard/src/helpers/settings.ts

```diff
@@ -1,9 +1,32 @@
 import { AsyncStorage } from 'react-native'
-
 export interface Settings {
     apiUrl: string
+    hasLiveDevMenu: boolean
+}
+
+/* 
```

Comment:
I think we would benefit from being able to persist objects instead of strings.

Reply:
it's using `json.parse` behind the scenes so knock yourself out!